udata/core/dataset/api.py

```diff
@@ -30,15 +32,36 @@
     upload_fields,
 )
 from .models import (
-    Dataset, Resource, FollowDataset, Checksum, License, UPDATE_FREQUENCIES
+    Dataset, Resource, FollowDataset, Checksum, License, UPDATE_FREQUENCIES,
+    CommunityResource
 )
 from .permissions import DatasetEditPermission
-from .forms import ResourceForm, DatasetForm
+from .forms import ResourceForm, DatasetForm, CommunityResourceForm
 from .search import DatasetSearch
 
 ns = api.namespace('datasets', 'Dataset related operations')
 search_parser = api.search_parser(DatasetSearch)
-
+parser = api.parser()
```

Comment:
As there is multiple parsers in this file, maybe it can have a more explicit name like ``community_resources_parser``